packages/atomic/src/components/search/result-template-components/quickview-preview-bar/quickview-preview-bar.ts

```diff
@@ -1,6 +1,44 @@
 import type {HighlightKeywords} from '../atomic-quickview-modal/atomic-quickview-modal';
```

Comment:
We should do a bit of refactoring / moving files around for quickview-related code as we work on the migration.

- The `HighlightKeywords` type is used by the `atomic-quickview-modal`, the `quickview-sidebar`, and the `quickview-preview-bar`. It should be declared in a separate `quickview-types` module (or something like that).
- The `atomic-quickview-modal` (which is the only code that uses `buildQuickviewPreviewBar` and `QuickviewSidebar`) is leveraged in both the Atomic search and insight use cases. We should move all of this code to commons.

I don't want to bloat this PR with this so: 

- [ ] Create Jira for said refactoring